src/router/paths.ts

```diff
@@ -0,0 +1,14 @@
+export const paths = {
+  root: '/',
+  login: '/admin-panel/login',
+  majors: '/majors',
+  majorsById: '/majors/:_id',
+  staff: '/staff',
+  staffById: '/staff/:_id',
+  news: '/news',
+  newsById: '/news/:_id',
+  ectsDegree: '/ects',
+  ectsMajor: 'ects/:degree/',
+  ectsYear: 'ects/:degree/:major',
+  ectsSubjects: 'ects/:degree/:major/:year',
```

Comment:
it would be better to make routing nested per module, but we can leave it for another task

Reply:
PLEASE LEAVE IT,  GO OVER